src/http/routing.rs

```diff
@@ -1031,8 +1031,8 @@ impl Route {
         api!("/applications/{}/guilds/{}/commands/permissions", application_id, guild_id)
     }
 
-    pub fn stage_instances() -> String {
-        api!("/stage-instances").to_string()
+    pub fn stage_instances() -> &'static str {
```

Comment:
It is inconsistent with others methods in this file

Reply:
no, it's not [Route::gateway](https://docs.rs/serenity/0.10.9/serenity/http/routing/enum.Route.html#method.gateway)